### PR TITLE
settings, global: Replace FILE setting type syntax

### PR DESCRIPTION
**Pull request by sirainen**
DOV-7085

### CODE REVIEW

File: docs/core/plugins/mail_crypt.md [hunk 1]
```diff
@@ -325,7 +325,7 @@ passdb static {
   password = pass
   fields {
     crypt_global_public_key_file = ecpubkey.pem
-    crypt_global_private_key/main/private_key = <content of ecprivkey.pem>
+    crypt_global_private_key/main/private_key_file = inline:<content of ecprivkey.pem>
```
Comment (cmouse) [inline docs/core/plugins/mail_crypt.md:328]: does this need to be base64 encoded because of newlines?

Comment (sirainen) [reply docs/core/plugins/mail_crypt.md:328]: You tell me :) You're the one who wrote the original documentation saying this, I've just kept modifying the syntax.

Comment (sirainen) [reply docs/core/plugins/mail_crypt.md:328]: But currently we don't support multi-line values in config-rewrite, so I guess so. So `base64-encoded content of ecprivkey.pem`?

Comment (sirainen) [reply docs/core/plugins/mail_crypt.md]: I added a new commit with some other changes too.

**Pull request merged**